ui/raidboss/data/05-shb/dungeon/the_grand_cosmos.txt

```diff
@@ -85,7 +85,7 @@ hideall "--sync--"
 1026.8 "Black Flame" sync /:Lugus:475D:/
 1030.3 "Otherworldly Heat" sync /:Lugus:475C:/
 1037.8 "Captive Bolt" sync /:Lugus:4764:/
-1048.2 "Mortal Flame" sync /:Lugus:4759:/ window 20,0
+1048.2 "Mortal Flame" sync /:Lugus:4759:/ window 30,0
```

Comment:
Honestly, at this rate, can you add a `--sync--` line for this ability starting and have it sync all the way back to t=1000?